### PR TITLE
feat: capped minter v2 pause

### PR DESCRIPTION
**Pull request by marcomariscal**

### CODE REVIEW

File: l2-contracts/foundry.toml [hunk 1]
```diff
@@ -9,6 +9,7 @@
     "@openzeppelin/contracts-upgradeable/=lib/openzeppelin-contracts-upgradeable/contracts/",
     "@openzeppelin/foundry-upgrades/=lib/openzeppelin-foundry-upgrades/src/",
     "@murky/=lib/murky/",
+    "src/=src/",
```
Comment (alexkeating) [inline l2-contracts/foundry.toml:12]: Remove

File: l2-contracts/test/ZkCappedMinterV2.t.sol [hunk 2]
```diff
@@ -91,6 +102,118 @@ contract Mint is ZkCappedMinterV2Test {
...
+    vm.prank(cappedMinterAdmin);
+    cappedMinter.pause();
+
+    // Remove PAUSER_ROLE from admin
+    vm.prank(cappedMinterAdmin);
+    cappedMinter.revokeRole(PAUSER_ROLE, cappedMinterAdmin);
+
+    vm.expectRevert(_formatAccessControlError(cappedMinterAdmin, PAUSER_ROLE));
+    vm.prank(cappedMinterAdmin);
+    cappedMinter.unpause();
```
Comment (alexkeating) [inline l2-contracts/test/ZkCappedMinterV2.t.sol:215]: Missing }

Comment (marcomariscal) [reply l2-contracts/test/ZkCappedMinterV2.t.sol:209]: https://github.com/ScopeLift/zk-governance/pull/22/commits/15568faa3b98b01977d99369b174af8d3f07e82f

File: l2-contracts/foundry.toml (hunk 1 above)
Comment (marcomariscal) [reply l2-contracts/foundry.toml]: https://github.com/ScopeLift/zk-governance/pull/22/commits/7b734162fe0be3fdd1216b01db0780fc2f2123f1

**Pull request merged**